### PR TITLE
Add multiply functionality and test

### PR DESCRIPTION
**Pull request by Machine-Control**
resolves #1

### CODE REVIEW

File: multiply.py [hunk 1]
```diff
@@ -0,0 +1,3 @@
+def multiply(x, y):
+    return x + y
+
```
Comment (EDSmith-shef) [inline multiply.py]: Newline required

File: multiply.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+def multiply(x, y):
+    return x + y
```
Comment (EDSmith-shef) [inline multiply.py]: Should be x * y

**Review by EDSmith-shef — CHANGES_REQUESTED** [review]
Thanks for PR, just a few changes needed.

**Review by EDSmith-shef — CHANGES_REQUESTED** [review]
Newline needed

**Comment by willf-training** [comment]
@EDSmith-shef The `multiply.py` file needs to be moved to the `pythoncalculator` subdirectory.

**Pull request merged**